### PR TITLE
Use port

### PR DESCRIPTION
**Pull request by ManuelHentschel**
See also https://github.com/ManuelHentschel/VSCode-R-Debugger/pull/49
Adds support to send responses and events via tcp socket.

### CODE REVIEW

**Comment by renkun-ken** [comment]
It looks like eval in debug console is somehow broken:

image

**Comment by ManuelHentschel** [comment]
Thanks for finding that issue!

**Comment by renkun-ken** [comment]
It works now. Thanks!

File: NAMESPACE [hunk 1]
```diff
@@ -17,14 +18,18 @@ export(.vsc.getCompletion)
...
 export(.vsc.listVarInfo)
+export(.vsc.listenOnPort)
 export(.vsc.onError)
 export(.vsc.print)
 export(.vsc.removeVarInfo)
 export(.vsc.resetVarInfo)
 export(.vsc.sendToVsc)
 export(.vsc.setBreakpoints)
 export(.vsc.setStoredBreakpoints)
+export(getSkipFromBottom)
```
Comment (renkun-ken) [inline NAMESPACE:32]: Are these two functions necessary to export?

Comment (ManuelHentschel) [reply NAMESPACE:32]: Nope, just forgot to remove them after debugging, thanks.

Comment (ManuelHentschel) [reply NAMESPACE:32]: Fixed in https://github.com/ManuelHentschel/vscDebugger/pull/67

**Pull request merged**